src/libraries/System.Collections/src/System/Collections/Generic/OrderedDictionary.cs

```diff
@@ -809,8 +809,8 @@ public void SetAt(int index, TKey key, TValue value)
             }
 
             // The key doesn't match that index. If it exists elsewhere in the collection, fail.
-            uint _ = 0, collisionCount = 0;
-            if (IndexOf(key, ref _, ref collisionCount) >= 0)
+            uint hashCode = 0, collisionCount = 0;
```

Comment:
Just curious why do we use uint for collision count? In general we use signed everywhere unless we need the full range.

Reply:
This just copies what `Dictionary<TKey, TValue>` does. I believe it uses uint just to make it very clear there's no possibility of overflow.